src/lib/drupal/lovell/tests/staticProps.test.ts

```diff
@@ -1,6 +1,46 @@
+// Mock declarations
+jest.mock('@/lib/drupal/staticProps', () => ({
+  fetchSingleStaticPropsResource: jest.fn(),
+  getDefaultStaticPropsResource: jest.fn(),
+}))
+
+jest.mock('../utils', () => ({
+  isLovellTricareSlug: (slug) => slug === lovellTricareSlug,
+  isLovellVaSlug: (slug) => slug === lovellVaSlug,
+  getLovellVariantOfBreadcrumbs: (breadcrumbs) => breadcrumbs,
+  getLovellVariantOfUrl: (url) => url,
+  getOppositeChildVariant: (variant) =>
+    variant === 'tricare' ? 'va' : 'tricare',
+  isLovellBifurcatedResource: () => false,
+}))
```

Comment:
Probably should add the rest of the real exports just in case.
```suggestion
jest.mock('../utils', () => ({
  ...jest.requireActual('../utils'),
  isLovellTricareSlug: (slug) => slug === lovellTricareSlug,
  isLovellVaSlug: (slug) => slug === lovellVaSlug,
  getLovellVariantOfBreadcrumbs: (breadcrumbs) => breadcrumbs, // Don't need this anymore
  getLovellVariantOfUrl: (url) => url, // Nor this
  getOppositeChildVariant: (variant) => // Probably this neither?
    variant === 'tricare' ? 'va' : 'tricare',
  isLovellBifurcatedResource: () => false,
}))
```

Also probably a good idea to do this for the other `jest.mock()` statements here.